samples/snippets/csharp/VS_Snippets_Remoting/System.Net.Dns/CS/dnsnewmethods.cs

```diff
@@ -22,6 +22,17 @@ public static void DoGetHostEntry(string hostname)
         }
 //</Snippet1>
 
+//<Snippet4>
+        public static void DoGetHostEntry(IPAddress address)
+        {
+            IPHostEntry host;
+
+            host = Dns.GetHostEntry(address);
+
+            Console.WriteLine("GetHostEntry({0}) returns HostName: {1}", address, host.HostName);
```

Comment:
Using interpolated strings is preferable here:
`Console.WriteLine($"GetHostEntry({address}) returns HostName: {host.HostName}");`